src/ol/DataTile.js

```diff
@@ -148,13 +148,13 @@ class DataTile extends Tile {
    * @return {import('./size.js').Size} Tile size.
    */
   getSize() {
-    if (this.size_) {
-      return this.size_;
-    }
     const imageData = asImageLike(this.data_);
```

Comment:
This avoids having to accurately specify tile sizes when using oversized tiles.  But that might make it incompatible with gutter unless the tiles have a gutter property.

Reply:
This has been moved to #16299.